dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -38,12 +38,12 @@ def report_abuse_pop_up
         return head :forbidden
       end
 
-      name = current_user&.name || "User not signed in"
-      email = current_user&.email || "User not signed in"
-      age = current_user&.age || "User not signed in"
+      name = current_user&.name
+      email = current_user&.email
+      age = current_user&.age
       abuse_url = CDO.studio_url(params[:abuse_url]) # reformats url
 
-      send_abuse_report(name, email, age, abuse_url)
+      send_abuse_report(name, email, age, abuse_url) if email
```

Comment:
Should we have some logging so we know how often signed out users are using this feature? Maybe not necessary if we're planning to follow up with a longer term fix soon

Reply:
Yeah I'd like to have a quick chat with support / product to see whether they want these reports or not, it isn't mentioned in [the spec](https://docs.google.com/document/d/1AYpsW1zfTNmlX38AyrSef7MmnYJTSA0KR7jihhkjZYk/edit?pli=1#heading=h.dltzor8mqxmn) that covered the initial work here as far as I can tell.

Reply:
(sorry, to answer your question -- I'd like to just get this immediate fix out, then answer longer term questions about signed out reports)